prospector.yml

```diff
@@ -40,6 +40,7 @@ pep257:
     - D106  # Missing docstring in public nested class
 
     # pydocstyle
+    - D403  # First word of the first line should be properly capitalized ('Github', not 'GitHub')
```

Comment:
The later in this example is actually their branding, so I don't understand this comment?

Reply:
This comment is the error message that prospector produces. I'm pasting the comment here to understand why I'm ignoring this particular code.

In this case, `D403` doesn't allow a docstring starting with `GitHub` string probably because it's CamelCase.